charts/ingress-nginx/Chart.yaml

```diff
@@ -24,4 +24,4 @@ maintainers:
 name: ingress-nginx
 sources:
 - https://github.com/kubernetes/ingress-nginx
-version: 4.9.0
```

Comment:
Remove this

Reply:
Hi @strongjz 
Revisions have been made. Please review.